package.json

```diff
@@ -25,13 +28,25 @@
     "internet of things",
     "serverless.com"
   ],
+  "jest": {
+    "testEnvironment": "node",
+    "testRegex": "(/tests/.*|(\\.|/)(test|spec))\\.js$",
+    "useStderr": true,
+    "setupFilesAfterEnv": [
+      "<rootDir>/tests/setup-tests.js"
+    ],
+    "verbose": true
```

Comment:
could you add under jest: `"testPathIgnorePatterns": ["tests/utils", "tests/setup-tests.js"],` to avoid testing unnecessary files